app/Rules/AlbumIDListRule.php

```diff
@@ -17,13 +17,10 @@ public function passes($attribute, $value): bool
 			return false;
 		}
 		$albumIDs = explode(',', $value);
-		if (!is_array($albumIDs) || count($albumIDs) === 0) {
-			return false;
-		}
 		$idRule = new AlbumIDRule(false);
 		$success = true;
 		foreach ($albumIDs as $albumID) {
-			$success &= $idRule->passes('', $albumID);
+			$success = $success && $idRule->passes('', $albumID);
```

Comment:
Argh! Recently when we (@nagmat84?) were fixing `|=` I was thinking that I've seen more of such code in Lychee but search returned no results. Now I know why...

Reply:
:D